GW2EIEvtcParser/EncounterLogic/Convergences/ConvergenceLogic.cs

```diff
@@ -0,0 +1,34 @@
+﻿using GW2EIEvtcParser.EIData;
+using GW2EIEvtcParser.ParsedData;
+using static GW2EIEvtcParser.ArcDPSEnums;
+using static GW2EIEvtcParser.EncounterLogic.EncounterCategory;
+
+namespace GW2EIEvtcParser.EncounterLogic;
+
+internal abstract class ConvergenceLogic : FightLogic
+{
+    protected ConvergenceLogic(int triggerID) : base(triggerID)
+    {
+        ParseMode = ParseModeEnum.FullInstance;
```

Comment:
I'm thinking we should introduce Instanced50, to be inline with Raids/Strikes/Fractals